### DISCUSSION TITLE
How to Add Hindi Language ?

### DISCUSSION THREAD

**Discussion by ghost** [Open-Shell/Open-Shell-Menu#841] [General]
Hi,
i want to contribute to open-shell by adding the language hindi . is this possible ? are there any was to add languages ?

**Comment by Gaurav-Original-ClassicShellTester** [comment]
See this topic: http://www.classicshell.net/forum/viewtopic.php?f=20&t=1344

**Comment by ghost** [comment]
Thanks, let me see the post
